site/_apps/quasar-hacker-news.txt

```diff
@@ -0,0 +1,28 @@
+---
+weight: 5
+title:  'Quasar Hacker News 1.0'
+github-title: 'hnpwa-quasar'
+libraries: 
+	- name: 'Quasar Framework'
+    - name: 'Vue'
+    - name: 'vue-router'
+module-bundling: 'Webpack'
+service-worker: 'Application Shell + data caching with SWPrecachePlugin'
+perfomance-patterns: 'none'
+server-side-rendering: 'no'
+api: 'Hacker News Firebase API'
+hosting: 'Github pages'
+other-details: 'Used the quasar framework for layout and shell'
+authors: 
+    - name: 'Jona Frank'
+lighthouse: '91/100'
+interactive-em: '9.457s'
+interactive-faster-3g: '6.287s'
+lighthouse-link: https://googlechrome.github.io/lighthouse/viewer/?gist=00aed89c357bad82c285cc5f98349212
```

Comment:
Looking at the performance scores here for Time-To-Interactive over 3G, this application is unfortunately not meeting the loads in <5s requirements of the specification (currently takes 8.3s to get interactive).

![image](https://user-images.githubusercontent.com/110953/42111706-31f26b66-7b9a-11e8-96dd-ddd54db08c6b.png)